### PR TITLE
Adding CDN Code

### PR DESCRIPTION
**Pull request by shreya-jaiswal37**
Please review CDN code.

### CODE REVIEW

File: cloudfront/cdn_code/variables.tf [hunk 1]
```diff
@@ -0,0 +1,268 @@
+variable "cloudfront_distributions" {
+  type = map(object({
+    #    aliases             = optional(list(string))
```
Comment (ashutoshyadav66) [inline cloudfront/cdn_code/variables.tf]: why aliases is commented on module

File: cloudfront/cdn_code/variables.tf [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+      function_association = optional(list(object({
+        event_type   = string
+        function_arn = string
+      })), [])
+
+    })), {})
+
+    viewer_certificate = optional(object({
+      # acm_certificate_arn            = optional(string)
```
Comment (ashutoshyadav66) [inline cloudfront/cdn_code/variables.tf]: why acm_certificate_arn is commented

File: cloudfront/terraform.tfvars [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            http_port              = 80
+            origin_protocol_policy = "http-only"
+          }
+        }
+      }
+    }
+    default_cache_behavior = {
+      "default_cache_behavior" = {
+        target_origin_id         = "CustomOrigin"
+        cache_policy_id          = "658327ea-f89d-4fab-a63d-7e88639e58f6"
```
Comment (ashutoshyadav66) [inline cloudfront/terraform.tfvars]: if you creating cache and request policy please use it

File: cloudfront/cdn_code/variables.tf (hunk 1 above)
Comment (shreya-jaiswal37) [reply cloudfront/cdn_code/variables.tf]: that was done by-mistake sir, please check now.

File: cloudfront/cdn_code/variables.tf (hunk 2 above)
Comment (shreya-jaiswal37) [reply cloudfront/cdn_code/variables.tf]: as discussed, I had updated this also.

File: cloudfront/terraform.tfvars (hunk 3 above)
Comment (shreya-jaiswal37) [reply cloudfront/terraform.tfvars]: sure sir, I had updated the code, please check once

**Pull request closed without merging**